gulp/tasks/build.js

```diff
@@ -5,6 +5,7 @@ var gulp = require( 'gulp' );
 gulp.task( 'build',
   [
     'styles',
+    'ieStyles',
```

Comment:
I thought about it but I wasn't sure since we typically wouldn't want it running locally during changes triggered by `gulp watch` due to run time increases, but maybe it should? What's everyone think?